lib/ansible/plugins/callback/__init__.py

```diff
@@ -502,6 +503,20 @@ def v2_on_any(self, *args, **kwargs):
         self.on_any(args, kwargs)
 
     def v2_runner_on_failed(self, result, ignore_errors=False):
+        # type: (TaskResult, bool) -> None
```

Comment:
This doesn't need to be a comment-based annotation. It is now possible to use native annotations without such hacks.